Challenge03/css/flor.css

```diff
@@ -0,0 +1,211 @@
+* {
+  font-family: Arial, sans-serif
+}
+
+/* Estilo generales */
+html {
+  color: whitesmoke;
+  background: #111111;
+}
+
+.page {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Arial, sans-serif;
+}
+
+/* Estilos específicos para dispositivos de pantalla grande */
+.header {
+  height: 80px;
+  background-color: #2b1f1f;
+  color: white;
+  border-radius: 10px;
+  opacity: 0.8;
+}
+
+.header__title {
+  text-align: center;
+  width: 100%;
+  line-height: 80px;
+}
+
+a {
+  color: white;
+  text-decoration: none;
+}
+
+
+.nav {
+  margin-top: 25px;
+  text-align: center;
+  height: 50px;
+  background-color: #666;
+  color: white;
+  display: flex;
+  align-items: center;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: space-around;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+}
+
+.nav__item {
+  margin: 0 100px;
+}
+
+.nav__link {
+  color: white;
+  text-decoration: none;
+  font-size: 20px;
+  font-weight: bold;
+  text-transform: uppercase;
+  transition: color 0.2s ease;
+  padding: 10px;
+}
+
+.nav__link:hover {
+  color: #ccc;
+}
+
+.main {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  grid-gap: 20px;
+  margin-top: 12vh;
+}
+
+.section {
+  text-align: justify;
+  min-height: 300px;
+  max-height: 500px;
+  background-color: #a02525;
+  padding: 20px;
+  box-sizing: border-box;
+  overflow: auto;
+  border-radius: 10px;
+}
+
+/* Estilos para la sección de subgéneros */
+#subgenres-section {

```

Comment:
I noticed that you are not using this class in your HTML file. You could delete it if you don't need to keep it.